http_check/README.md

```diff
@@ -46,6 +46,7 @@ See the [sample http_check.d/conf.yaml][3] for a full list and description of av
 | `include_content`                | When set to `true`, the check includes the first 500 characters of the HTTP response body in notifications. The default value is `false`.                                                                                                        |
 | `collect_response_time`          | By default, the check collects the response time (in seconds) as the metric `network.http.response_time`. To disable, set this value to `false`.                                                                                                 |
 | `tls_verify`                     | Instructs the check to validate the TLS certificate of services when reaching to `url`.                                                                                                                                                          |
+| `tls_retrieve_non_validated_cert`| If `tls_verify` is set to `false`, setting this to `true` allows the certificate to be examined for an expiration date.                                                                                                                      |
 | `tls_ignore_warning`             | If `tls_verify` is set to `true`, it disables any security warnings from the SSL connection.                                                                                                                                                     |
 | `tls_ca_cert`                    | This setting allows you to override the default certificate path as specified in `init_config`                                                                                                                                                   |
 | `check_certificate_expiration`   | When `check_certificate_expiration` is enabled, the service check checks the expiration date of the SSL certificate. **Note**: This causes the SSL certificate to be validated, regardless of the value of the `tls_verify` setting. |
```

Comment:
Small nit, could we move your `tls_retrieve_non_validated_cert` under this `check_certificate_expiration` option since the two are related? I would also update your description to include that `check_certificate_expiration` should be set to true to make the connection explicit.

Reply:
Will do

Reply:
updated